airbyte-integrations/connectors/source-mailjet-sms/source_mailjet_sms/spec.yaml

```diff
@@ -0,0 +1,15 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Mailjet Sms Spec
+  type: object
+  required:
+    - token
+  additionalProperties: true
+  properties:
+    token:
+      type: string
+      description: >- 
+        Your access token. See <a
+        href="https://dev.mailjet.com/sms/reference/overview/authentication">here</a>.
+      airbyte_secret: true
```

Comment:
The API allows you to use start date and end date, are you planning to add those parameters?